src/ui/transactions/usePositionsRefetchInterval.ts

```diff
@@ -0,0 +1,61 @@
+import { useStore } from '@store-unit/react';
+import { useEffect, useState } from 'react';
+import { Store } from 'store-unit';
+import { localTransactionsStore } from './transactions-store';
+
+const latestTxTimeStore = new Store(0);
+
+localTransactionsStore.on('change', (state) => {
+  const knownLatest = latestTxTimeStore.getState();
+  // in localTransactionsStore transactions are sorted reverse-chronologically
+  // so the newest item is the first item
+  const latestTx = state.at(0);
+  if (latestTx) {
+    if (latestTx.timestamp > knownLatest) {
+      latestTxTimeStore.setState(latestTx.timestamp);
+    }
+  }
+});
+
+const TWO_MINUTES = 1000 * 60 * 2;
+const URGENT_REFETCH_INTERVAl = 4000; // 4 seconds
+
+function useNow(enabled: boolean) {
+  const [now, setNow] = useState(() => Date.now());
+  useEffect(() => {
+    if (enabled) {
+      // doesn't really need to match URGENT_REFETCH_INTERVAl,
+      // but also we don't need to refresh more frequently than that
+      const refreshInterval = URGENT_REFETCH_INTERVAl;
+      const id = setInterval(() => {
```

Comment:
Is it ok that we will update `now` only 4 sec after `enabled` was set to `true`?

Reply:
I think so